libs/application/templates/social-insurance-administration/medical-and-rehabilitation-payments/src/lib/messages.ts

```diff
@@ -125,6 +135,41 @@ export const medicalAndRehabilitationPaymentsFormMessage: MessageDir = {
         'Hefur þú nýtt rétt þinn til sjúkradagpeninga frá stéttarfélagi?',
       description: 'Have you used your union sick pay entitlement?',
     },
+    unionSickPayUnionDescriptionTitle: {
+      id: 'marp.application:general.information.union.sick.pay.union.description.title',
+      defaultMessage: 'Skráning stéttarfélags',
+      description: 'Union registration',
+    },
+    unionSickPayUnionSelectTitle: {
+      id: 'marp.application:general.information.union.sick.pay.union.select.title',
+      defaultMessage: 'Stéttarfélag',
+      description: 'Union',
+    },
+    unionSickPayUnionSelectPlaceholder: {
+      id: 'marp.application:general.information.union.sick.pay.union.select.placeholder',
+      defaultMessage: 'Veldu stéttarfélag',
+      description: 'Select union',
+    },
+    unionSickPayDidEndDate: {
+      id: 'marp.application:general.information.union.sick.pay.did.end.date',
+      defaultMessage: 'Hvenær lauk rétti þínum til sjúkradagpeninga?',
+      description: 'When did your right to sick pay end?',
+    },
+    unionSickPayDoesEndDate: {
+      id: 'marp.application:general.information.union.sick.pay.does.end.date',
+      defaultMessage: 'Hvenær lýkur rétti þínum til sjúkradagpeninga?',
+      description: 'When does your right to sick pay end?',
```

Comment:
Update text to match `unionSickPayTitle`
```suggestion
      description: 'When does your entitlement to union sick pay end?',
```

Reply:
Fixed